crud/users.py

```diff
@@ -202,3 +203,73 @@ async def set_new_role(uid, new_role, user_role, db: AsyncSession):
     await db.commit()
     return {'status': 'success',
             'message': f'User {uid} successfully changed to {new_role.role}.'}
+
+
+@handle_errors
+async def set_reminder(user_id, reminder_set, db: AsyncSession):
+    """
+    Function to set the reminders on or off for the user.
+
+    Returns:
+        Message of the transaction.
+    """
+    query = sa.select(User).where(User.id == user_id)
+    result = await db.execute(query)
+    user = result.scalar()
+    if user.daily_reminder == reminder_set and reminder_set == "yes":
+        raise HTTPException(status_code=200,
+                            detail='Reminders are already enabled for this user')
+    if user.daily_reminder == reminder_set and reminder_set == "no":
+        raise HTTPException(status_code=200,
+                            detail='Reminders are already disabled for this user')
+
+    reminders_config = (
+        sa.update(User).where(User.id == user_id).values(daily_reminder=reminder_set)
+    )
+    await db.execute(reminders_config)
+    await db.commit()
+    return {'status': 'success', 'message': 'Reminders successfully configured.'}
+
+
+@handle_errors
+async def send_reminders(user_role, db: AsyncSession):
+    """
+    Function to send the reminders to users with this function
+
+    Returns:
+        Message of the transaction.
+    """
+    if user_role != 'admin':
+        raise HTTPException(status_code=403,
+                            detail=NOT_AUTHORIZED)
+    query = text("CREATE TEMPORARY TABLE temp_user_tasks AS SELECT u.email, "
```

Comment:
Maybe it`s better to get list of open todos in code instead:
get list of open todos using sqlalchemy and then then send it to users